services/cd-service/pkg/repository/transformer.go

```diff
@@ -190,7 +192,11 @@ func (c *CreateApplicationVersion) Transform(ctx context.Context, fs billy.Files
 			return "", err
 		}
 	}
-
+	if c.Configuration != "" {
+		if err := util.WriteFile(fs, fs.Join(appDir, "config.json"), []byte(c.Configuration), 0666); err != nil {
```

Comment:
in that case I would suggest properties format: https://pkg.go.dev/github.com/magiconair/properties#section-readme

Reply:
My first thought was to make it `yaml` or `json` but I chose `json` because we already using it for env config, so if I should change it what is the best way to do that?  Assuming we have N features we can't create a simple file for each of them? what would you suggest?

Reply:
I would make it an actual text file. Like we do for the source_commit_id. This gets deduplicated quite well in git and is also easy to read without parsing.

Reply:
Having just one value per file makes it pretty hard to add new features. We want to make it easier.